storage/persistent-storage/persistent-storage-vsphere.adoc

```diff
@@ -0,0 +1,63 @@
+[id="persistent-storage-using-vsphere"]
+= Persistent Storage Using VMware vSphere
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-vsphere
+
+toc::[]
+
+{product-title} supports VMWare vSphere's Virtual Machine Disk (VMDK). You 
+can provision your {product-title} cluster with persistent storage using 
+VMware vSphere. Some familiarity with Kubernetes and VMware vSphere is 
+assumed.
+
+The Kubernetes persistent volume framework allows administrators to 
+provision a cluster with persistent storage and gives users a way to 
+request those resources without having any knowledge of the underlying 
+infrastructure. vSphere VMDK can be provisioned dynamically.
+
+Persistent volumes (PV) are not bound to a single project or namespace; 
+they  can be shared across the {product-title} cluster.
+Persistent volume claims (PVC) are specific to a project or namespace and 
+can be requested by users.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+.Prerequisites
+Before creating PVs using vSphere, ensure your
+{product-title} cluster meets the following requirements:
+
+* {product-title} must first be configured for vSphere.
+* Each node host in the infrastructure must match the vSphere VM name.
+* Each node host must be in the same resource group.
+* A VMDK must first exist, created in one of the following methods:
+** Created using `vmkfstools`. Access ESX through a secure shell (SSH) and 
```

Comment:
I don't think this should start with "Created" since you already lead in saying "created in one of the following methods". I would just start with "Using".